src/pages/home/report/ReportActionCompose.js

```diff
@@ -66,18 +72,24 @@ class ReportActionCompose extends React.Component {
         this.setIsFocused = this.setIsFocused.bind(this);
         this.focus = this.focus.bind(this);
         this.comment = props.comment;
+        this.autoFocusEnabled = getPlatform() === 'desktop' || (getPlatform() === 'web' && !canUseTouchScreen());
```

Comment:
I'm pretty sure you've noticed by now, but this will need to be split up so we can use file extensions to determine whether to focus the input. Also, I don't think we need the distinction of desktop or web?

This name is a little verbose but I think it explains what we are trying to figure out here...

`canFocusInputOnScreenFocus()`

Then 

```js
// canAutoFocusTextInput/index.js
return () => !canUseTouchScreen();
```
```js
/// canAutoFocusTextInput/index.native.js
return () => false;
```